src/jiradeps/config.py

```diff
@@ -12,6 +12,8 @@
 [server]
 url: https://example.local/jira
 username: test
+# Alternatively, you can specify a personal access token.
```

Comment:
I'd state explicitly that it is actually alternatively to the username, and that the token is overriding the UN/PW